docs/recipes/WriteToContractWriteAsyncButton.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 2
+title: Write to contract with writeAsync button
+description: Learn how to create a button that executes the writeAsync function to interact with a smart contract.
+---
+
+# Write to a Contract with `writeAsync` button
+
+This recipe shows how to implement a button that allows users to interact with a smart contract by executing the `writeAsync` function. By following this guide, you can create a user interface for writing data to a contract.
+
+In this recipe you will use [useScaffoldContractWrite](/hooks/useScaffoldContractWrite) Scaffold ETH-2 hook, and [ParseEther viem utility](https://viem.sh/docs/utilities/parseEther.html#parseether).
```

Comment:
```suggestion
This recipe shows how to implement a button that allows users to interact with a smart contract by executing the `writeAsync` function returned by [useScaffoldContractWrite](/hooks/useScaffoldContractWrite). By following this guide, you can create a user interface for writing data to a contract.
```

Maybe we remove the line "In this recipe you will use..."  and maybe link to `useScaffoldContractWrite` in description itself just to keep it short and also consistent with other sections (since other don't have "In  recipe ..."  line)

Reply:
Ty @technophile-04! Totally forgot to delete that part from it 🙌